src/lib/snipTagContent.ts

```diff
@@ -16,7 +16,7 @@ export function snipScriptAndStyleTagContent(source: string): string {
         const indexes: [number, number][] = [];
         let match = null;
         while ((match = regex.exec(source)) != null) {
-            if (!source.substr(match.index, 10).startsWith('<!--')) {
+            if (!source.slice(match.index, match.index + 10).startsWith('<!--')) {
```

Comment:
This code (both the old and the new version) looks really weird to me. Surely this is equivalent to

```suggestion
            if (source.slice(match.index, match.index + 4) !== '<!--') {
```

?